### PR TITLE
[ONNX] Remove the INT64_MAX magic numbers

### PR DESCRIPTION
**Pull request by justinchuby**
Remove the magic numbers in symbolic opsets and use a INT64_MAX  global instead.

### CODE REVIEW

File: torch/onnx/symbolic_opset10.py [hunk 1]
```diff
@@ -345,7 +346,7 @@ def _slice(
         if (
             len(starts) == 1
             and starts[0] == 0
-            and ends[0] == 9223372036854775807
+            and ends[0] == _INT64_MAX
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset10.py:349]: Is it only appearing in this file? Should `_INT64_MAX` reside in `_constants.py`?

**Review by BowenBao — APPROVED** [review]

File: torch/onnx/symbolic_opset10.py (hunk 1 above)
Comment (justinchuby) [reply torch/onnx/symbolic_opset10.py]: Sure! Moved to _constants

**Pull request closed without merging**